lib/check/check.py

```diff
@@ -145,22 +149,30 @@ def import_check(check_path: str) -> ModuleType:
     return lib
 
 
-def set_output_options(quiet):
+def set_output_options(quiet: bool, output_modes: list):
     global output_options
     output_options = Output_From_Options(
-        is_quiet=quiet
+        is_quiet=quiet,
+        output_modes=output_modes
         # set input options here
     )
     return output_options
 
 
-def run_check(check):
+def run_check(check, audit_info, output_options):
     print(
         f"\nCheck Name: {check.checkName} - {Fore.MAGENTA}{check.serviceName}{Fore.YELLOW}[{check.severity}]{Style.RESET_ALL}"
     )
     logger.debug(f"Executing check: {check.checkName}")
     findings = check.execute()
-    report(findings, output_options)
+    organizations_info = Organizations_Info(
+        account_details_email="test@verica.io",
+        account_details_name="washemen",
+        account_details_arn="arn:aws:iam::account:test/test",
+        account_details_org="ou-washemen",
+        account_details_tags="tag1:likisomawei",
+    )
+    report(findings, output_options, audit_info, organizations_info)
```

Comment:
I think it would be better to split the `run_check` function in two separate functions, one to run the check and another to report the execution findings.

We need to be able to test this logic separately.

What do you think?

Reply:
It is divided now, a call for filling orgs info is made, right now that function only instantiates an empty object to be passed to `report` function call